azurerm/resource_arm_mssql_elasticpool.go

```diff
@@ -119,39 +120,70 @@ func resourceArmMsSqlElasticPool() *schema.Resource {
 			},
 
 			"elastic_pool_properties": {
-				Type:     schema.TypeList,
-				Computed: true,
-				MaxItems: 1,
+				Type:       schema.TypeList,
+				Computed:   true,
+				MaxItems:   1,
+				Deprecated: "All properties herein have been move to the top level",
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"state": {
-							Type:     schema.TypeString,
-							Computed: true,
+							Type:       schema.TypeString,
+							Computed:   true,
+							Deprecated: "This property has been moved to the top level",
 						},
 
 						"creation_date": {
-							Type:     schema.TypeString,
-							Computed: true,
+							Type:       schema.TypeString,
+							Computed:   true,
+							Deprecated: "This property has been moved to the top level",
 						},
 
 						"max_size_bytes": {
-							Type:     schema.TypeInt,
-							Computed: true,
+							Type:       schema.TypeInt,
+							Computed:   true,
+							Deprecated: "This property has been moved to the top level",
 						},
 
 						"zone_redundant": {
-							Type:     schema.TypeBool,
-							Computed: true,
+							Type:       schema.TypeBool,
+							Computed:   true,
+							Deprecated: "This property has been moved to the top level",
 						},
 
 						"license_type": {
-							Type:     schema.TypeString,
-							Computed: true,
+							Type:       schema.TypeString,
+							Computed:   true,
+							Deprecated: "This property has been moved to the top level",
 						},
 					},
 				},
 			},
 
+			"state": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"creation_date": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"max_size_bytes": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+
+			"zone_redundant": {
+				Type:     schema.TypeBool,
+				Computed: true,
+			},
+
+			"license_type": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
```

Comment:
I can’t think of when this field would be likely to be used from within Terraform - should we remove it?